Assets/Scripts/SelfDestruct.cs

```diff
@@ -0,0 +1,13 @@
+﻿using UnityEngine;
```

Comment:
How is this used? Are health pickups generated at world generation or do they constantly spawn...?

Reply:
It's placed on the particle effect instantiated when you pickup a medical canister so that the gameobject is destroyed once the particle effect ends